keyboards/ergodash/rev1/keymaps/m47ch4n/keymap.c

```diff
@@ -0,0 +1,226 @@
+#include QMK_KEYBOARD_H
+#include "keymap_jp.h"
+
+extern keymap_config_t keymap_config;
```

Comment:
This line is not actually needed, and should be removed. 

```suggestion
```

Reply:
I appreciate your code review!
I have fixed code according your suggestions https://github.com/qmk/qmk_firmware/pull/6673/commits/ba9fc0ee6c5b34f922e018f8860296ac288d467a.
Thank you.